tools/BUILD.bazel

```diff
@@ -0,0 +1,30 @@
+load("@bazelrc-preset.bzl", "bazelrc_preset")
+load("@buildifier_prebuilt//:rules.bzl", "buildifier", "buildifier_test")
+
+buildifier(
```

Comment:
can we add `tools` to export-ignore like https://github.com/dzbarsky/rules_rs/blob/main/.gitattributes#L4 ?
that way we don't end up with unbuildable targets in the module

Reply:
Sure